front/types/core/core_api.ts

```diff
@@ -405,6 +369,119 @@ function topKSortedDocuments(
   return result;
 }
 
+export type CoreAPILlmChatContentTextResponse = { value: string };
```

Comment:
typescript types of the new endpoint. 

I didn't see any swagger or else so I wrote them manually (with AI)